dev/src/dev.clj

```diff
@@ -307,3 +317,15 @@
      (mt/with-driver (:engine db#)
        (mt/with-db db#
          ~@body))))
+
+(defn set-log-level!
+  "Set logging level for a namespace you're in right now."
+  ([level]
+   (set-log-level! (ns-name *ns*) level))
+  ([ns level]
+   (mt/set-ns-log-level! ns level)))
+
+(defmacro p
+  "#p, but to use in pipelines like `(-> 1 inc dev/p inc)`"
+  [form]
+  (hashp/p* form))
```

Comment:
Can you write a little more about this? I have no idea what `#p` is so I don't really know what this does at all

Reply:
Does it print something?